packages/lavapack/package.json

```diff
@@ -16,6 +16,7 @@
     "json-stable-stringify": "^1.0.1",
     "lavamoat-core": "^14.0.0",
     "readable-stream": "^3.6.0",
+    "resolve": "^1.22.1",
```

Comment:
resolve uses https://socket.dev/npm/package/is-core-module internally. If we don't need other functionality of resolve elsewhere, let's minimize dependency.

Reply:
See https://github.com/LavaMoat/LavaMoat/pull/470/files#r1135433710